pages/location/index.html

```diff
@@ -48,7 +48,7 @@
     <page-header assetsUrl="../../assets"></page-header>
 
     <div class="pages-content">
-        <back-anchor assetsUrl="../../assets"></back-anchor>
+        <back-anchor href="/" assetsUrl="../../assets"></back-anchor>
```

Comment:
The works for local dev, but unfortunately, it doesn't work for the prod. The reason is because the root URL on prod is ```https://github.com/AR-js-org/studio```